conferences/models.py

```diff
@@ -2,7 +2,10 @@
 from django.db.models import Count, F
 from django.utils.translation import ugettext as _
 from django.core.exceptions import ValidationError
-from datetime import timedelta, datetime
+from datetime import (
+    datetime,
+    timedelta
+)
```

Comment:
why do you put it in four lines? it would be a lot nicer as oneliner. Brackets in imports are usefull only when import is too long to fit in one line...